src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
+}) { 
+    const classNames = 
+    (italic? 'italic ' : '') + 
+    (bold? 'font-bold ' : '') + 
+    (underline? 'underline ' : '') + 
```

Comment:
(nit) these are ternary operators, I would put a space between the boolean and the resolutions.
```suggestion
    (italic ? 'italic ' : '') + 
    (bold ? 'font-bold ' : '') + 
    (underline ? 'underline ' : '') + 
```
Otherwise, it makes these seem like either an optional prop on a class, or an optional chaining operation... It can get confusing

Reply:
Agree with this. It took me a little longer than I'd like to admit to realize that it was an operator so definitely add the space